### DISCUSSION TITLE
Merge Cells

### DISCUSSION THREAD

**Discussion by Chiliyago** [minhhungit/ConsoleTableExt#20] [Ideas]
First, thank you for this library. It may be just the one I have been looking for.  Have you thought about the possibility of merging cells or adding additional space above, below, left or right of the text in a cell or row?

**Comment by minhhungit** [comment]
hi @Chiliyago  At this time you can add padding for only left and right for each cells using **WithPaddingLeft** and **WithPaddingRight**

I have thought about merging cell but I have no idea how to implement it yet
